src/lib/data.json

```diff
@@ -0,0 +1,219 @@
+{
+    "supportedYears": [
+      {
+        "value": "2018",
+        "label": "2018"
+      },
+      {
+        "value": "2019",
+        "label": "2019"
+      },
+      {
+        "value": "2020",
+        "label": "2020"
+      },
+      {
+        "value": "2021",
+        "label": "2021"
+      },
+      {
+        "value": "2022",
+        "label": "2022"
+      }
+    ],
+    "personType": [
+      {
+        "value": "single",
+        "label": "Single"
+      },
+      {
+        "value": "separate",
+        "label": "Separate"
+      },
+      {
+        "value": "together",
+        "label": "Together"
+      },
+      {
+        "value": "head",
+        "label": "Head"
+      }
+    ],
+    "yearToTax": {
+      "2018": {
+          "percents": [0.1, 0.12, 0.22, 0.24, 0.32, 0.35, 0.37],
+          "single": [0, 9525, 38700, 82500, 157500, 200000, 500000],
+          "separate": [0, 9525, 38700, 82500, 157500, 200000, 300000],
+          "together": [0, 19050, 77400, 165000, 315000, 400000, 600000],
+          "head": [0, 13600, 51800, 82500, 157500, 200000, 500000],
+          "ss": 128400,
+          "budget": 6300000000000,
+          "budgetPercents": [0.1678, 0.1646, 0.1577, 0.1103, 0.0868, 0.0848, 0.0685, 0.0326, 0.0209, 0.0205, 0.0184, 0.0134, 0.0122, 0.0121, 0.009, 0.006, 0.0045, 0.0029, 0.0069],
+          "standardDeduction": {
+            "single": 12000,
+            "separate": 12000,
+            "together": 24000,
+            "head": 18000
+          }
+      },
+      "2019": {
+          "percents": [0.1, 0.12, 0.22, 0.24, 0.32, 0.35, 0.37],
+          "single": [0, 9700, 39475, 84200, 160725, 204100, 510300],
+          "seperate": [0, 9700, 39475, 84200, 160725, 204100, 306175],
+          "ss": 132900,
```

Comment:
It seems like you're missing entries for "together and "head" here. Is it possible to add them?